plugin.go

```diff
@@ -2,31 +2,32 @@
 package plasmactlbump
```

Comment:
- [ ] Print a warning during propagation when version of resource is not a bump commit but another commit
      echo " " >> interaction/softwares/roles/rainloop/templates/Dockerfile.j2
      git add interaction/softwares/roles/rainloop/templates/Dockerfile.j2 && git commit -m "whatever"
      compose and sync
```
WARNING  resource interaction__softwares__rainloop is marked as updated, but version in build (f1aedfd81d06e) and artifact (f1aedfd81d06e) are identical, skipping.
> 
WARNING  latest commit of interaction__softwares__rainloop (6bccfea21fdeb) is not a bump commit.
```

Reply:
New message is better: ` WARNING  latest commit of interaction__softwares__rainloop (f1aedfd81d06e) is not a bump commit.` 

however it should print commit hash of domain repo, not the one from artifact